.gitignore

```diff
@@ -60,6 +60,4 @@ typings/
 .env
 
 # next.js build output
-.next
-
-package-lock.json
+.next
```

Comment:
This is funny. I guess I should recursively add the newline-action to the newline-action repo itself.